lib/cdo/redshift.rb

```diff
@@ -28,12 +28,13 @@ class PostgreSQLQueryError < StandardError; end
 
   def initialize
     require 'pg'
-
-    port = 5439
-    options = ''
-    tty = ''
-    dbname = 'dashboard'
-    @conn = PG::Connection.new(CDO.redshift_host, port, options, tty, dbname, CDO.redshift_username, CDO.redshift_password)
+    @conn = PG::Connection.new(
+      host: CDO.redshift_host,
+      port: 5439,
+      dbname: 'dashboard',
+      user: CDO.redshift_username,
+      password: CDO.redshift_password
+    )
```

Comment:
This change isn't strictly necessary; both the 1.0 and 1.3 versions of this method support passing options in a variety of ways ([see docs](https://www.rubydoc.info/gems/pg/PG/Connection#new-class_method)). But this way we don't need to declare empty variables for `options` and `tty`, and it's a little DRYer